recipes/less-learn-mpi/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2021 Ilker Birbil @ UvA
```

Comment:
The license is already included in the PyPI source distribution so you can probably remove it from here.